libr/debug/p/native/xnu/xnu_debug.c

```diff
@@ -1398,7 +1398,7 @@ RList *xnu_dbg_maps(RDebug *dbg, int only_modules) {
 	//bool contiguous = false;
 	//ut32 oldprot = UT32_MAX;
 	//ut32 oldmaxprot = UT32_MAX;
-	char buf[1024];
+	char buf[1152];
```

Comment:
arbitrary constants are not good. better use local `char *buf = r_str_newf` when needed below